packages/core/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -17,52 +25,35 @@ import * as vscode from 'vscode'
 import { spawnSync } from 'child_process' // Consider using ChildProcess once we finalize all spawnSync calls
 import AdmZip from 'adm-zip'
 import globals from '../../shared/extensionGlobals'
-import {
-    CodeTransformJavaSourceVersionsAllowed,
-    CodeTransformMavenBuildCommand,
-    telemetry,
-} from '../../shared/telemetry/telemetry'
+import { CodeTransformMavenBuildCommand, telemetry } from '../../shared/telemetry/telemetry'
 import { codeTransformTelemetryState } from '../../amazonqGumby/telemetry/codeTransformTelemetryState'
-import {
-    calculateTotalLatency,
-    javapOutputToTelemetryValue,
-    JDKToTelemetryValue,
-} from '../../amazonqGumby/telemetry/codeTransformTelemetry'
+import { calculateTotalLatency, javapOutputToTelemetryValue } from '../../amazonqGumby/telemetry/codeTransformTelemetry'
 import { MetadataResult } from '../../shared/telemetry/telemetryClient'
 import request from '../../common/request'
-import { ToolkitError } from '../../shared/errors'
 
 // log project details silently
 export async function validateAndLogProjectDetails() {
-    let reason,
-        result,
-        codeTransformLocalJavaVersion,
-        codeTransformPreValidationError = undefined
+    let reason = undefined
+    let javaVersion = undefined
     try {
         const openProjects = await getOpenProjects(true)
         const validProjects = await validateOpenProjects(openProjects, true)
         if (validProjects.size > 0) {
             const firstKey = validProjects.keys().next().value
-            const firstModuleEntry = validProjects.get(firstKey)
-            codeTransformLocalJavaVersion = JDKToTelemetryValue(
-                firstModuleEntry
-            ) as CodeTransformJavaSourceVersionsAllowed
+            javaVersion = validProjects.get(firstKey)
         }
     } catch (err: any) {
-        result = MetadataResult.Fail
-        reason = err?.code
-        codeTransformPreValidationError = err?.name
```

Comment:
Since I changed the way we do Error handling, `err.message` is really what we want to access